defs/globals.js

```diff
@@ -63,6 +63,7 @@ export interface StepProps {
   showSkipButton?: boolean;
   spotlightClicks?: boolean;
   spotlightPadding?: number;
+  spotlightTarget: string | HTMLElement;
```

Comment:
I've marked it as optional but didn't change the name.
This is because this PR doesn't change element on which tooltip is positioned, but change spotlighted area. So `spotlightTarget` is a target only to "hole" other than element pointed by `target`.

Reply:
Probably my description of #468 was not clear - it was an idea, but when I tried to implement it, I realized that it will be better do add new prop for spotlight target.

Reply:
This is confusing.
Did you test it? How? With scroll?

Reply:
I was thinking about your approach and I don't think is feasible.

The spotlight is used in the scrolling calculation and if the user selects an element that isn't in the same viewport of the target, the tour will be broken. Or if the spotlightTarget isn't related to the step's target the UX will be terrible.
Did you try to set a different target for the tooltip instead? It will be more logical for users.